### PR TITLE
Remove LGPL dependencies from vcpkg port

### PR DESCRIPTION
**Pull request by Nopey**
getopt-win32: removed
libxmp: updated to the latest git revision, which has been relicensed as MIT.
Latest release libxmp 4.6.0 was before the relicense, so I'm just gonna avoid that one.

Obsoletes PR #721

### CODE REVIEW

File: vcpkg-ports/argtable3/portfile.cmake [hunk 1]
```diff
@@ -5,14 +5,13 @@ vcpkg_from_github(
...
         "Remove-DllMain.patch"
 )
 
 set(ARGTABLE3_REPLACE_GETOPT ON)
-if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-   set(ARGTABLE3_REPLACE_GETOPT OFF)
-endif()
+#if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+#   set(ARGTABLE3_REPLACE_GETOPT OFF)
+#endif()
```
Comment (mrannanj) [inline vcpkg-ports/argtable3/portfile.cmake:14]: Why comment out instead of removing these?

Comment (Nopey) [reply vcpkg-ports/argtable3/portfile.cmake]: no reason, removed them.

**Review by mrannanj — APPROVED** [review]

**Comment by Vagabond** [comment]
Why don't we need getopt anymore?

**Comment by Nopey** [comment]
We're using argtable3's own implementation by setting ARGTABLE3_REPLACE_GETOPT

**Pull request merged**